TravisCI builds are broken

```
$ flake8 homu
homu/server.py:195:5: F841 local variable 'ex' is assigned to but never used
homu/server.py:652:21: F841 local variable 'ex' is assigned to but never used
The command "flake8 homu" exited with 1.
```
And python 3.3 is no longer supported by some package, so that job is totally busted.

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,7 +1,6 @@
 sudo: false
 language: python
 python:
-  - 3.3
   - 3.4
   - 3.5
   - 3.6

--- a/homu/server.py
+++ b/homu/server.py
@@ -192,7 +192,7 @@ def callback():
             'client_secret': g.cfg['github']['app_client_secret'],
             'code': code,
         })
-    except Exception as ex:
+    except Exception as ex:  # noqa
         logger.warn('/callback encountered an error '
                     'during github oauth callback')
         # probably related to https://gitlab.com/pycqa/flake8/issues/42
@@ -649,7 +649,7 @@ def buildbot():
                                         props['buildnumber'],
                                         step_name,)
                         res = requests.get(url)
-                    except Exception as ex:
+                    except Exception as ex:  # noqa
                         logger.warn('/buildbot encountered an error during '
                                     'github logs request')
                         # probably related to

